### DISCUSSION TITLE
Tasmota32Webcam 13.3.0 not working with OV5640 lense ( ESP32Cam and new OV5640 ) Board

### DISCUSSION THREAD

**Discussion by smartboart** [arendst/Tasmota#20511] [Q&A] [answered]
Hello, i was able to get the new OV5640_V1_2  ESP32 Cam Board running with a OV2640 lense. Therefore a found a working template.
But the OV5640 Lense is not running with Tasmota 13.3.0webcam. Even not with the older ESP32- Cam board. 
I think it is not supporting the better lense... Found information about 12MHz needed for xclk ... Any chance to get a new projekt or update working with this lense?
![296908134-96637bdb-bd13-404e-a43e-847ae37870d3](https://github.com/arendst/Tasmota/assets/38385805/ab52ee2f-5e83-45e4-a04a-b2950de3bf35)

![296907802-e836dbf0-4886-4168-b5a4-ceddc0588686](https://github.com/arendst/Tasmota/assets/38385805/9d18402b-46d1-4e76-97af-6280354d65e3)

https://github.com/blakadder/templates/issues/1836

**Comment by Jason2866** [comment, marked as answer]
As designed. Not supported. To save flash space driver is not in IDF precompiled libs.
The OV2640 is the only supported one.
Official espressif example code https://github.com/espressif/esp-iot-solution/tree/master/examples/camera

**Reply by smartboart** [reply]
Hi, thanks for feedback.. is there a chance that somebody will make it possible? I am not so deep in the code to do so.
But if necessary i can test things ...

**Reply by Jason2866** [reply]
It is a design decision for Tasmota not to support other cams than the OV2640.
Needs changes in the Tasmota Arduino framework, not only in Tasmota code. So no, will not be done.
If you want to use the OV5640 Tasmota is the wrong choice.
